project.yml

```diff
@@ -12,18 +12,33 @@ targets:
         DEVELOPMENT_TEAM: Kaoru Muta
         INFOPLIST_FILE: QiitaChecker/Info.plist
         PRODUCT_BUNDLE_IDENTIFIER: com.muta.QiitaChecker
+    dependencies:
+      - carthage: RxSwift.framework
+        linkType: static
+      - carthage: RxCocoa.framework
+        linkType: static
+      - carthage: RxRelay.framework
```

Comment:
Due to storing rx framework under the file path`Carthage/build/iOS/Static`framework, build failed

Reply:
Anyway, move rx framework to parent directory, it worked